deepparse/embeddings_models/embeddings_model.py

```diff
@@ -14,7 +15,7 @@ def __init__(self, verbose: bool = True) -> None:
             print("Loading the embeddings model")
 
     @abstractmethod
-    def __call__(self, word: str) -> ndarray:
+    def __call__(self, word: Union[str, List[str]]) -> Union[ndarray, List[ndarray]]:
```

Comment:
Since we use Magnitude in batch, it's a list, the other two models create a list with the ndarray later on in the vectorizing (https://github.com/GRAAL-Research/deepparse/blob/c9f9e37b5356eb8d06980fe02eae2bfee49ea7d5/deepparse/vectorizer/fasttext_vectorizer.py#L34)

Reply:
I think you should change this function to take a list parameter and output a Numpy array of shape [batch_size, len(list)].

Reply:
7735f9a